src/broker/broker_config.c

```diff
@@ -83,23 +83,42 @@
 #define	TRUE	1
 #define	FALSE	0
 
+#define MAX_NUM_BROKER_FILES	4
```

Comment:
How about changing variable name to MAX_NUM_CACHED_BROKER_FILES ?
And, would four cached broker files be enough ?

Reply:
Your comment is accepted.
Thank you for your comment!